numpy/random/_generator.pyi

```diff
@@ -0,0 +1,245 @@
+        self, a: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def laplace(
+        self,
+        loc: _ArrayLikeFloat_co = ...,
+        scale: _ArrayLikeFloat_co = ...,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def gumbel(
+        self,
+        loc: _ArrayLikeFloat_co = ...,
+        scale: _ArrayLikeFloat_co = ...,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def logistic(
+        self,
+        loc: _ArrayLikeFloat_co = ...,
+        scale: _ArrayLikeFloat_co = ...,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def lognormal(
+        self,
+        mean: _ArrayLikeFloat_co = ...,
+        sigma: _ArrayLikeFloat_co = ...,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def rayleigh(
+        self, scale: _ArrayLikeFloat_co = ..., size: Union[None, _ShapeLike] = ...
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def wald(
+        self,
+        mean: _ArrayLikeFloat_co,
+        scale: _ArrayLikeFloat_co,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def triangular(
+        self,
+        left: _ArrayLikeFloat_co,
+        mode: _ArrayLikeFloat_co,
+        right: _ArrayLikeFloat_co,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    # Complicated, discrete distributions:
+    def binomial(
+        self, n: _ArrayLikeInt_co, p: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[int, ndarray[Any, dtype[int64]]]: ...
+    def negative_binomial(
+        self, n: _ArrayLikeFloat_co, p: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[int, ndarray[Any, dtype[int64]]]: ...
+    def poisson(
+        self, lam: _ArrayLikeFloat_co = ..., size: Union[None, _ShapeLike] = ...
+    ) -> Union[int, ndarray[Any, dtype[int64]]]: ...
+    def zipf(
+        self, a: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[int, ndarray[Any, dtype[int64]]]: ...
+    def geometric(
+        self, p: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[int, ndarray[Any, dtype[int64]]]: ...
+    def hypergeometric(
+        self,
+        ngood: _ArrayLikeInt_co,
+        nbad: _ArrayLikeInt_co,
+        nsample: _ArrayLikeInt_co,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[int, ndarray[Any, dtype[int64]]]: ...
+    def logseries(
+        self, p: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[int, ndarray[Any, dtype[int64]]]: ...
+    # Multivariate distributions:
+    # TODO: Really need  1-d array like floating and 2-d array-like floating. Using Sequence[float] ??
```

Comment:
Same as the other two: Just ignore the dimensionality for now and reserve it as future work.

Reply:
Thanks.